[0m12:31:05.509010 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f1ee290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f7bda50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f7bc990>]}


============================== 12:31:05.514003 | b0e17666-b319-47ed-9c70-2fcd9f302d78 ==============================
[0m12:31:05.514003 [info ] [MainThread]: Running with dbt=1.5.0
[0m12:31:05.514473 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '.', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m12:31:14.632281 [error] [MainThread]: Encountered an error:

[0m12:31:14.640475 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/cli/requires.py", line 86, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/cli/requires.py", line 71, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/cli/main.py", line 459, in init
    results = task.run()
              ^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/task/init.py", line 286, in run
    project_name = self.get_valid_project_name()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/task/init.py", line 243, in get_valid_project_name
    name = click.prompt("Enter a name for your project (letters, digits, underscore)")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/click/termui.py", line 166, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/click/termui.py", line 149, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m12:31:14.643461 [debug] [MainThread]: Command `dbt init` failed at 12:31:14.642422 after 9.18 seconds
[0m12:31:14.644596 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f7cad90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f2a0ed0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ed18890>]}
[0m12:31:14.645355 [debug] [MainThread]: Flushing usage events
[0m12:31:39.949763 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f2cfc50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f5e2f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f5c5fd0>]}


============================== 12:31:39.954511 | 51eadf40-eec2-4deb-aa5b-7ea31d72eeb2 ==============================
[0m12:31:39.954511 [info ] [MainThread]: Running with dbt=1.5.0
[0m12:31:39.954984 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '.', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m12:32:36.338204 [debug] [MainThread]: Starter project path: /Users/shazeltine/miniconda/envs/dbt-citibike/lib/python3.11/site-packages/dbt/include/starter_project
